src/pages/infrastructure.js

```diff
@@ -113,7 +113,7 @@ const emergingProjects = [
     logoUrl: 'https://github.com/rprinz08/hBPF',
     name: 'hBPF',
     logo: hbpfLogo,
-    title: 'hBPF - eBPF in hardware',
+    title: 'eBPF in hardware',
```

Comment:
Nit: All others use title case.

```suggestion
    title: 'eBPF in Hardware',
```